### PR TITLE
ENH changes for split sysroot

### PR DESCRIPTION
**Pull request by beckermr**
We have to patch the repo data for the old binutils to make them conflict with the old sysroot before we can merge this.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #24

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
We shouldn't merge this to `main` just yet because this would bring in another conflicting `sysroot`.

**Comment by beckermr** [comment]
So I am pretty out of my depth here. Right now this simply rearranges the sysroot we have, or does it do something more?

**Comment by isuruf** [comment]
`gcc_linux-64` packages we have currently vendor the sysroot. `sysroot_linux-64` is not used by any package at the moment.

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf what is the ordering of merging this? I see a gfortran failure and am wondering if we need to merge the compiler change first?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ outputs:
     requirements:
       build:
       host:
+        - sysroot_{{ target_platform }}
       run:
```
Comment (isuruf) [inline recipe/meta.yaml:40]: We need to add the following, merge this, merge the compiler and then remove this.
```suggestion
      run:
        - sysroot_{{ target_platform }}
```

Comment (beckermr) [reply recipe/meta.yaml:40]: Ahhhhhh. That's a fun game!

Comment (beckermr) [reply recipe/meta.yaml:40]: I see. We need to boostrap the whole thing with the sysroot around. Makes sense.

**Comment by beckermr** [comment]
woops! done!

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
Alright. Seems like this is ready to roll?

**Pull request merged**